consensus/polybft/bridge/bridge_manager.go

```diff
@@ -150,6 +157,106 @@ func (b *bridgeEventManager) Start(runtimeConfig *config.Runtime) error {
 
 	b.tracker = tracker
 
+	relayer, err := createBridgeTxRelayer(b.config.bridgeCfg.JSONRPCEndpoint, b.logger)
+	if err != nil {
+		return fmt.Errorf("failed to initialize bridge external client. Error: %w", err)
+	}
+
+	b.externalClient = *relayer.Client()
+
+	return nil
+}
+
+// internalChainRollbackHandler manages rollback logic for batches that have not been
+// successfully executed in the internal chain
+func (b *bridgeEventManager) internalChainRollbackHandler(blockNumber *big.Int, dbTx *bolt.Tx) error {
+	if err := b.createRollbackBatches(blockNumber, b.externalChainID, b.internalChainID, dbTx); err != nil {
+		b.logger.Error("could not create a rollback batches", "err", err)
+
+		return nil
```

Comment:
Shouldn't we propagate an error here?

```suggestion
		return err
```